docs/third-party-developers/extensibility/data-store/payment.md

```diff
@@ -27,6 +27,136 @@ with. We do not encourage extensions to dispatch actions onto this data store ye
 
 ## Selectors
 
+### isPaymentPrestine
+
+Queries if the status is `pristine` or not.
```

Comment:
Nitpicky, but could we reword to: "Returns whether the status is `pristine`.

Feel free to ignore I just think it sounds a little better without the "or not". This applies to the rest of the doc too.